scrapper.py

```diff
@@ -25,39 +36,135 @@ class Crawler:
     """
     Crawler implementation
     """
+
     def __init__(self, seed_urls, max_articles: int):
-        pass
+        self.seed_urls = seed_urls
+        self.total_max_articles = max_articles
+        self.urls = []
 
     def _extract_url(self, article_bs):
-        pass
+        cells = article_bs.find_all("div", class_='col-lg-3 col-sm-6')
+        for cell in cells:
+            if len(self.urls) >= self.total_max_articles:
+                break
```

Comment:
can you rewrite this function without break point?

Reply:
solved this problem